packages/@ourworldindata/types/src/dbTypes/MultiDimXChartConfigs.ts

```diff
@@ -0,0 +1,9 @@
+export const MultiDimXChartConfigsTableName = "multi_dim_x_chart_configs"
+
+export type DbInsertMultiDimXChartConfig = {
+    multiDimId: number
```

Comment:
Since it's an auto-increment field, I think it makes sense to explicitly not have the `id`, because including it is invalid, and you must leave it out. Which is different from optional fields, like `updatedAt`.